test/unit/spec/OktaSignIn_spec.js

```diff
@@ -18,6 +18,7 @@ const itp = Expect.itp;
 Expect.describe('OktaSignIn initialization', function () {
   let signIn;
 
+  /* eslint jasmine/no-global-setup:0 */
```

Comment:
what is this change?

Reply:
I really don't know why I was getting lint errors about the `beforeEach`, I think it doesn't recognize `Expect.describe` but not sure why it is effecting me, maybe version of eslint.

Reply:
Only the top level we need to use `Expect.describe` otherwise `describe` is good enough.
Does it fail bacon/travis as well (without this overwrite)

Reply:
yes I was seeing the same failure on Bacon.

Reply:
https://bacon-go.aue1e.saasure.net/#!/commits/okta-signin-widget/sha/cad8dc85197e71ce3cee5a0d6a2e115818ed4ad6